internal/validation/v2alpha1/hosts.go

```diff
@@ -22,6 +25,30 @@ func validateHosts(parentAttributePath string, vsList networkingv1beta1.VirtualS
 	}
 
 	for hostIndex, host := range hosts {
+		if !helpers.IsFqdnHostName(string(*host)) {
+			if helpers.IsShortHostName(string(*host)) {
```

Comment:
```suggestion
			if helpers.IsShortHostName(string(*host)) {
```
First "if" seems not to be needed as the regex are mutually exclusive (short host prohibits `.` and FQDN requires it to exist)

Reply:
yes, they are mutually exclusive, no need to have nested ifs, will refactor